packages/braid-design-system/package.json

```diff
@@ -200,7 +200,7 @@
     "fs-extra": "^10.1.0",
     "html-validate": "^7.1.1",
     "playroom": "0.32.1",
-    "prettier": "^2.8.2",
+    "prettier": "^2.8.8",
```

Comment:
GitHub wouldn't let me merge because it thought `pnpm-lock.yaml` had conflicts (it didn't) so I made an unrelated change to update the lockfile.